src/Description.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import gql from "graphql-tag";
+import { graphql, compose } from "react-apollo";
+
+const updateUserQuery = gql`
+  mutation($id: ID!, $description: String!) {
+    updateUser(id: $id, description: $description) {
+      id
+    }
+  }
+`;
+
+const userQuery = gql`
+  query {
+    user {
+      id
+    }
+  }
+`;
+
+class Description extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: ""
+    };
+    this.updateUser = props.updateUser;
+  }
+
+  updateDB() {
+    const { description } = this.state;
+    const { id } = this.props.data.user
+    this.props.updateUser({ variables: { id, description } });
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
wondering if we can remove the empty wrapping `<div/>` ?

Reply:
I believe if you are returning multiple jsx elements you have to have a single wrapper...

Reply:
oh right! sorry i missed the other element thanks!